x-pack/legacy/plugins/siem/public/components/ml/api/get_ml_capabilities.ts

```diff
@@ -26,9 +26,11 @@ export const getMlCapabilities = async (
   headers: Record<string, string | undefined>
 ): Promise<MlCapabilities> => {
   try {
-    const response = await fetch('/api/ml/ml_capabilities', {
+    const response = await fetch(`${chrome.getBasePath()}/api/ml/ml_capabilities`, {
       method: 'GET',
+      credentials: 'same-origin',
       headers: {
+        credentials: 'same-origin',
```

Comment:
Don't think you want this here in `headers`?

Reply:
No I do not. Thank you.